app/controllers/reviews_controller.rb

```diff
@@ -31,8 +31,11 @@ def create
     @review.site = @site
     @review.user = current_user
     if @review.save
+      @review.site.reviewed
```

Comment:
consider an instance method on review that "wraps this behavior" and mimics the return values of `save`

```
if @review.issue

# in app/models/review
def review
  if @review.save
    #send email
  else
    return false
  end
end
```

Then, write correlating unit tests to ensure this behaves as expected